src/api/app/views/webui2/webui/projects/rebuild_times/show.html.haml

```diff
@@ -0,0 +1,32 @@
+:ruby
+  content_for(:content_for_head, javascript_include_tag('webui2/cm2/index-prjconf'))
+  @pagetitle = "Rebuild time for Project #{@project}"
+  @metarobots = 'noindex'
+
+.card
+  = render(partial: 'webui/project/tabs', locals: { project: @project })
+  .card-body
+    %h3= "Rebuildtime: #{distance_of_time_in_words(Time.now, Time.now - @rebuildtime)} (#{format_seconds(@rebuildtime)})"
```

Comment:
I think these strings aren't really best practise in HAML.

```
%h3
  Rebuildtime:
  = distance_of_time_in_words(Time.now, Time.now - @rebuildtime)} (#{format_seconds(@rebuildtime)})

Reply:
actually we were waiting some ask for a helper.. Nobody? :(

Reply:
> I think these strings aren't really best practise in HAML.
> 
> ```
> %h3
>   Rebuildtime:
>   = distance_of_time_in_words(Time.now, Time.now - @rebuildtime)} (#{format_seconds(@rebuildtime)})
> ```

Didn't work, because of the call to `format_seconds(@rebuildtime)` after.

Changed to:

```
%h3
  Rebuildtime: #{distance_of_time_in_words(Time.now, Time.now - @rebuildtime)} (#{format_seconds(@rebuildtime)})
```

What is inside each `#{}` should be a variable calculated in the controller.

Reply:
only because my editing was wrong. This should do (still untested though):

```
%h3
  Rebuildtime:
  = distance_of_time_in_words(Time.now, Time.now - @rebuildtime)
  (#{format_seconds(@rebuildtime)})

Reply:
to be clear: I'm not suggesting to block the PR because of this